### PR TITLE
Move Metadata from setup.py to setup.cfg

### LINKED ISSUE #25: Move Metadata from setup.py to setup.cfg
- Move metadata from `setup.py` to `setup.cfg`

### PR DESCRIPTION
**Pull request by ernavdeepkr**
Move Metadata from setup.py to setup.cfg
fixes #25 

Signed-off-by: Navdeep Kaur <ernavdeepkr@gmail.com>

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,3 +1,40 @@
 [flake8]
 ignore = E128,E811,W503,E203
 max-line-length = 100
+
+[metadata]
+name = linkstatus
+author = Nikhil Dhandre
+author_email = nik.digitronik@live.com
+license = GPLv3
```
Comment (digitronik) [inline setup.cfg]: If possible please include meta
```
license-file = LICENSE
```

File: setup.cfg [hunk 2]
```diff
@@ -1,3 +1,40 @@
...
+    Programming Language :: Python :: 3.7
+    Intended Audience :: End Users/Desktop
+    Intended Audience :: Developers
+    Environment :: Console
+
+[options]
+packages = find:
+zip_safe = False
+setup_requires = setuptools_scm
+install_requires = click; markdown; requests
```
Comment (digitronik) [inline setup.cfg]: Optional: I would try to write this in following way
```
install_requires = 
    click
    markdown
    requests
```

File: setup.cfg [hunk 3]
```diff
@@ -1,3 +1,40 @@
...
+include_package_data = True
+python_requires = >=3.5
+
+[options.entry_points]
+console_scripts = linkstatus=linkstatus.linkstatus:main
```
Comment (digitronik) [inline setup.cfg]: Optional: for more readability
```
console_scripts = 
    linkstatus = linkstatus.linkstatus:main
```

File: setup.py [hunk 4]
```diff
@@ -1,39 +1,3 @@
...
-    setup_requires=setup_requirements,
-    use_scm_version=True,
-    keywords=["linkstatus", "linkchecker", "link-checker", "markdown", "text", "linklint", "link"],
-    name="linkstatus",
-    packages=find_packages(include=["linkstatus"]),
-    url="https://github.com/pythonpune/linkstatus",
-    license="GPLv3",
-    zip_safe=False,
-)
+setuptools.setup()
```
Comment (digitronik) [inline setup.py]: Required:
we are using `scm_version`; so we need to pass it.
```
setuptools.setup(use_scm_version=True,)
```

File: setup.cfg (hunk 1 above)
Comment (ernavdeepkr) [reply setup.cfg]: Based upon https://setuptools.readthedocs.io/en/latest/setuptools.html#metadata
`license_file = LICENSE` added. This meta accepts an only string.

File: setup.cfg (hunk 3 above)
Comment (ernavdeepkr) [reply setup.cfg]: Done

File: setup.py (hunk 4 above)
Comment (ernavdeepkr) [reply setup.py]: didn't know where to mention. Done now.

File: setup.cfg (hunk 2 above)
Comment (ernavdeepkr) [reply setup.cfg]: Done

**Comment by ernavdeepkr** [comment]
Rebased with upstream and fixup.

**Pull request merged**